packages/qwik/src/server/v2-ssr-container.ts

```diff
@@ -421,6 +421,10 @@ class SSRContainer extends _SharedContainer implements ISSRContainer {
     this.write(rawHtml);
   }
 
+  commentNode(text: string) {
+    this.write('<!--' + text + '-->');
```

Comment:
I think these comment nodes will confuse the current implementation of `materializeFromVNodeData,` so we will need to update that.